docs/runtimes/mlrun_code_annotations.ipynb

```diff
@@ -0,0 +1,1250 @@
+       "      <td>0</td>\n",
+       "      <td>Oct 28 11:11:37</td>\n",
+       "      <td>completed</td>\n",
+       "      <td>my-fucntion-name</td>\n",
+       "      <td><div class=\"dictlist\">v3io_user=admin</div><div class=\"dictlist\">kind=</div><div class=\"dictlist\">owner=admin</div><div class=\"dictlist\">host=jupyter-75877f79b-dpgbh</div></td>\n",
+       "      <td></td>\n",
+       "      <td></td>\n",
+       "      <td><div class=\"dictlist\">return=hello from my-fucntion</div></td>\n",
+       "      <td></td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div></div>\n",
+       "  <div id=\"result1496b1fc-pane\" class=\"right-pane block hidden\">\n",
+       "    <div class=\"pane-header\">\n",
+       "      <span id=\"result1496b1fc-title\" class=\"pane-header-title\">Title</span>\n",
+       "      <span onclick=\"closePanel(this)\" paneName=\"result1496b1fc\" class=\"close clickable\">&times;</span>\n",
+       "    </div>\n",
+       "    <iframe class=\"fileview\" id=\"result1496b1fc-body\"></iframe>\n",
+       "  </div>\n",
+       "</div>\n"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "<b> > to track results use the .show() or .logs() methods  or <a href=\"https://dashboard.default-tenant.app.alonm32.iguazio-cd0.com/mlprojects/default/jobs/monitor/c139dc40c4ae4665a5c79dd3b9fcff8e/overview\" target=\"_blank\">click here</a> to open in UI</b>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "> 2021-10-28 11:11:37,733 [info] run executed, status=completed\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<mlrun.model.RunObject at 0x7f80207dbf90>"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "my_fucntion = code_to_function('my-fucntion-name', kind='job')\n",
+    "my_fucntion.run(name='my-fucntion-name', handler='handler', local=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Multi section function\n",
+    "You can use the `# mlrun: start-code` and `# mlrun: end-code` annotations multiple times in a notebook, the whole notebook will be scanned."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# mlrun: start-code multi-section-fucntion\n",
```

Comment:
do we need to add `multi-section-fucntion` after the start/end?

Reply:
we need some function name for this example because nameless annotations are already used in the 1st example in this notebook, using nameless annotations again will merge the code from both examples

Reply:
but this is confusing, if u say its not needed and u just do it because u already have it in the notebook, we may just want to settle for an example (e.g. in markdown) or a separate notebook vs make ppl think they need to add the named func part

Reply:
When people going towards the more advanced stuff like multi-section function I prefer them to be explicit and specify the function name, to make it clear this is just the function name (and not some magic string or something) I suggest to change it to `multi-section-function-name`. In addition, to make it clear that it's possible to use multi-section with nameless annotations, explain it in the first sentence (after the `you can use...`)
@yaronha @AlonMaor14 WDYT ?

Reply:
im ok as long as its very clear that the name is optional, we should also highlight that things like imports etc. should be placed in each function

Reply:
sounds good to me